addons/binding/org.openhab.binding.freebox/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,233 @@
+ 				<description>Token generated by the Freebox server</description>
+ 				<required>true</required>
+ 			</parameter>
+ 
+	 		<parameter name="refreshInterval" type="integer">
+ 				<label>Refresh Interval</label>
+ 				<description>The refresh interval in seconds which is used to poll given Freebox Server.</description>
+ 				<default>30</default>
+ 				<required>true</required>
+ 			</parameter>
+ 			
+ 	 		<parameter name="refreshPhoneInterval" type="integer">
+ 				<label>Refresh Interval</label>
+ 				<description>The refresh interval in seconds which is used to poll given Freebox Server.</description>
+ 				<default>2</default>
+ 				<required>true</required>
+ 			</parameter>
+ 
+ 		</config-description>
+    </thing-type>
+    
+    <channel-type id="fwversion">
+        <item-type>String</item-type>
+        <label>Firmware Version</label>
+        <description>Version of the Freebox Server Firmware</description>
+    </channel-type>
+    
+    <channel-type id="uptime">
+        <item-type>Number</item-type>
+        <label>Server uptime</label>
+        <description>Time since last reboot of the Freebox Server</description>
+    </channel-type>
+    
+	<channel-type id="restarted">
+		<item-type>Switch</item-type>
+		<label>Just restarted</label>
+		<description>Has the Freebox server have restarted during the last poll period</description>
+	</channel-type>
+	
+	<channel-type id="restarted">
+		<item-type>Switch</item-type>
+		<label>Just restarted</label>
+		<description>Has the Freebox server have restarted during the last poll period</description>
+	</channel-type>
+	
+	<channel-type id="reboot">
+		<item-type>Switch</item-type>
+		<label>Reboot Freebox</label>
+		<description>Reboots the Freebox server</description>
+	</channel-type>
+	
+	<channel-type id="tempcpum">
+		<item-type>Number</item-type>
+		<label>CPUm Temperature</label>
+		<description>Actual measured CPU Marvell temperature of the Freebox Server</description>
+	</channel-type>
+	
+	<channel-type id="tempcpub">
+		<item-type>Number</item-type>
+		<label>CPUb Temperature</label>
+		<description>Actual measured CPU Broadcom (xDSL) temperature of the Freebox Server</description>
+	</channel-type>
+	
+	<channel-type id="tempswitch">
+		<item-type>Number</item-type>
+		<label>Switch Temperature</label>
+		<description>Actual measured switch temperature of the Freebox Server</description>
+	</channel-type>
+	
+	<channel-type id="fanspeed">
+		<item-type>Number</item-type>
+		<label>Fan Speed</label>
+		<description>Actual measured fan speed (rpm) of the Freebox Server</description>
+	</channel-type>
+	
+	<channel-type id="lcd_brightness">
+		<item-type>Number</item-type>
+		<label>Screen Brightness</label>
+		<description>Brightness level of the screen in percent</description>
+	</channel-type>
+	
+	<channel-type id="lcd_orientation">
+		<item-type>Number</item-type>
+		<label>Screen Orientation</label>
+		<description>Orientation de l'écran</description>
```

Comment:
Could you please translate the text to english? Note that you can provide french translations in the i18n folder.